### ISSUE TITLE
No longer working with new TypeScript version

### ISSUE THREAD

**Issue by DAmNRelentless** [lightyen/ts-paths-resolve-plugin#3]
I updated to TypeScript 4.4 and the plugin is no longer able to resolve the config it seems.

Versions:
```
"ts-loader": "^9.2.5",
"ts-node": "^10.2.1",
"ts-paths-resolve-plugin": "^5.2.3",
"typescript": "^4.4.2",
"webpack": "^5.51.1",
"webpack-cli": "^4.8.0"
```

Error:
```
[webpack-cli] Failed to load '/mnt/c/Users/Relentless/Documents/GitKraken/twitch_rlnt-bot/webpack.config.ts' config
[webpack-cli] Error: (!) [TsPathsResolvePlugin]: No inputs were found in config file 'tsconfig.json'. Specified 'include' paths were '["src/**/*"]' and 'exclude' paths were '["out"]'.
    at TsPathsResolvePlugin.getTsConfig (/mnt/c/Users/Relentless/Documents/GitKraken/twitch_rlnt-bot/node_modules/ts-paths-resolve-plugin/dist/index.js:93:19)
    at new TsPathsResolvePlugin (/mnt/c/Users/Relentless/Documents/GitKraken/twitch_rlnt-bot/node_modules/ts-paths-resolve-plugin/dist/index.js:73:37)    at Object.<anonymous> (/mnt/c/Users/Relentless/Documents/GitKraken/twitch_rlnt-bot/webpack.config.ts:27:19)
    at Module._compile (/mnt/c/Users/Relentless/Documents/GitKraken/twitch_rlnt-bot/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Module.m._compile (/mnt/c/Users/Relentless/Documents/GitKraken/twitch_rlnt-bot/node_modules/ts-node/src/index.ts:1310:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Object.require.extensions.<computed> [as .ts] (/mnt/c/Users/Relentless/Documents/GitKraken/twitch_rlnt-bot/node_modules/ts-node/src/index.ts:1313:12)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
```

TsConfig:
```
{
  "compilerOptions": {
    "incremental": false,
    "target": "ES2021",
    "module": "CommonJS",
    "rootDir": "src",
    "outDir": "out",
    "sourceMap": true,
    "removeComments": true,

    "strict": true,
    "noImplicitAny": true,
    "strictNullChecks": true,
    "strictFunctionTypes": true,
    "strictBindCallApply": true,
    "strictPropertyInitialization": true,
    "noImplicitThis": true,
    "exactOptionalPropertyTypes": true,
    "useUnknownInCatchVariables": true,
    "alwaysStrict": true,

    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "noImplicitReturns": true,
    "noFallthroughCasesInSwitch": true,
    "noUncheckedIndexedAccess": true,
    "noImplicitOverride": true,
    "noPropertyAccessFromIndexSignature": true,

    "moduleResolution": "node",
    "allowSyntheticDefaultImports": true,
    "esModuleInterop": true,

    "skipLibCheck": true,
    "forceConsistentCasingInFileNames": true,

    "baseUrl": "./src",
    "paths": {
      "@/*": ["*"],
      "@commands/*": ["./commands/*"],
      "@internals/*": ["./internals/*"],
      "@modules/*": ["./modules/*"],
      "@utils/*": ["./utils/*"]
    }
  },
  "include": ["src/**/*"]
}
```

**Comment by lightyen** [comment]
TypeScript has [breaking changes](https://github.com/microsoft/TypeScript/pull/44710) not mentioned in [blog](https://devblogs.microsoft.com/typescript/announcing-typescript-4-4-beta) between 4.3.5 and 4.4, and I don't know what happened.

I will ignore the error from `ts.parseJsonConfigFileContent()` in next version.

---

- Test:
```js
const ts = require("typescript")
console.log(ts.sys.readDirectory("", [".ts", ".tsx", ".d.ts"], undefined, ["src/**/*"]))
```

**Comment by lightyen** [comment]
Fixed in 5.3.1
